### PR TITLE
[14.0][MIG] sale_order_type: Migration to 14.0

### PR DESCRIPTION
**Pull request by ferran-S73**
Standard migration to version 14.0

### CODE REVIEW

**Comment by rousseldenis** [comment]
@ferran-73 Could you check travis ?

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[run]
```
Comment (rousseldenis) [inline .coveragerc:1]: Why are you modifying this ???

**Comment by rousseldenis** [comment]
@ferran-73 It seems that you didn't look into https://github.com/OCA/sale-workflow/issues/1274

That module is already done there. Could you review it ?

**Comment by ferran-S73** [comment]
> @ferran-73 It seems that you didn't look into #1274
> 
> That module is already done there. Could you review it ?

You're right, didn't find it. I'll close this pull request.

**Pull request closed without merging**